src/test_workflow/integ_test/integ_test_suite_opensearch.py

```diff
@@ -87,7 +88,8 @@ def __setup_cluster_and_execute_test_config(self, config: str) -> int:
         ) as (endpoints):
             os.chdir(self.work_dir)
             self.pretty_print_message("Running integration tests for " + self.component.name + " " + config)
-            return self.multi_execute_integtest_sh(endpoints, security, config)
+            script = ScriptFinder.find_integ_test_script(self.component.name, self.repo.working_directory)
```

Comment:
Why do we need to call the scriptfinder here?

Reply:
To make sure I separate the ftrepo clone for dashboards, Will change it back in the child class implementations.